lib/jekyll/admin/api_servlet.rb

```diff
@@ -0,0 +1,43 @@
+require "webrick"
+require "json"
+module Jekyll
+  module Admin
+    class ApiServlet < WEBrick::HTTPServlet::AbstractServlet
+      def initialize(server, site)
+        super(server)
+        @site = site
+      end
+
+      def do_GET(_request, response) # rubocop:disable Style/MethodName
+        hash = { :site => @site, :data => "Hello World" }
```

Comment:
This was just a dummy servlet to serve up the `/api` route while I was figuring out how to make GETs and POSTs work in WEBrick Servlets. Now it serves the purpose of the default api page (on `/api` as other endpoints are on `/api/*` and I have updated it to show some help text instead.